docs/how-to/build-docs.rst

```diff
@@ -0,0 +1,22 @@
+How to build the documentation
+******************************
+
+Create a virtual environment and activate it::
+
+    python3 -m venv docs/env
+    source docs/env/bin/activate
+
+Install the documentation requirements::
+
+    pip install -r requirements-doc.txt
```

Comment:
I tried to follow this locally but I got an error with `python-apt`; the command that worked for me is `pip install -r requirements-focal.txt -r requirements-doc.txt`

Reply:
I can't comment on that, the plain pip install -r requirements-doc.txt worked for me, but it could be that I have some global Python components or environment conditions that make it work.